### PR TITLE
Membership fail presenter

### PR DESCRIPTION
**Pull request by KaiNissen**

### CODE REVIEW

File: src/UseCases/ApplyForMembership/ApplyForMembershipRequest.php [hunk 1]
```diff
@@ -45,6 +46,15 @@ public function setMembershipType( string $membershipType ) {
 		$this->membershipType = $membershipType;
 	}
 
+	public function getShowMembershipTypeOption(): bool {
```
Comment (JeroenDeDauw) [inline src/UseCases/ApplyForMembership/ApplyForMembershipRequest.php]: This is presentation specific and (thus) not used by the UC, so should not be in here

Comment (KaiNissen) [inline src/UseCases/ApplyForMembership/ApplyForMembershipRequest.php]: Right. This is directly passed to the presenter now.

File: src/Presentation/Presenters/MembershipFormViolationPresenter.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	public function __construct( TwigTemplate $template ) {
+		$this->template = $template;
+	}
+
+	public function present( array $violations, ApplyForMembershipRequest $request, bool $showMembershipTypeOption ): string {
+		return $this->template->render(
+			[
+				'initialFormValues' => $this->getMembershipFormArguments( $request ),
+				'violatedFields' => $violations,
```
Comment (JeroenDeDauw) [inline src/Presentation/Presenters/MembershipFormViolationPresenter.php]: Like this the template binds to stuff in the Validators NS. Kinda iffy

**Comment by JeroenDeDauw** [comment]
+1

**Comment by KaiNissen** [comment]
removed `violatedFields`, rebased

**Pull request merged**